src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,69 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    @OneToMany
+    private List<Ticket> tickets;
```

Comment:
Consider specifying the `mappedBy` attribute in the `@OneToMany` annotation to define the bidirectional relationship with the `Ticket` entity. Additionally, it might be beneficial to use `CascadeType.ALL` or specific cascade types to manage the lifecycle of the `tickets` collection properly.